src/icon/wheelchair.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { PureComponent } from 'react'
+import cc from 'classcat'
+import isEmpty from 'lodash.isempty'
+
+import { color } from '_utils/branding'
+
+class WheelchairIcon extends PureComponent<Icon> {
+  static defaultProps: Icon = {
+    className: '',
+    iconColor: color.icon,
+    size: 24,
+    title: '',
+  }
+
+  render() {
+    const { className, iconColor, size, title } = this.props
+    return (
+      <svg
+        viewBox="0 0 100 100"
+        xmlns="http://www.w3.org/2000/svg"
+        className={cc(['kirk-icon', className])}
+        width={size}
+        height={size}
+        aria-hidden={isEmpty(title)}
+      >
+        {title && <title>{title}</title>}
+        <g fill={iconColor}>
+          <path
```

Comment:
Looks like the paths can be further optimized: https://jakearchibald.github.io/svgomg/

Reply:
Done. Thx.